git-and-github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+_This goes in your project's root directory under `.github/`_
+## Expected Behavior/Desired Use Case
+_If this is a bug, report what **should** be happening. If it's a feature, describe what this **should** implement._
+
+## Actual/Current Behavior
+_Describe what is currently happening._
+
+
+## Steps to Reproduce the Problem/Implement this feature
+  1.
+  2.
+  3.
+
+## Specifications
```

Comment:
Instead of "Specifications", how about changing this to "Environment" or "System Requirements" or "Prerequisites"?

Reply:
@stephengtuggy Good idea! I will push that up.